workflows/python/sdk/order-processor/util.py

```diff
@@ -0,0 +1,10 @@
+from dapr.conf import settings
+
+def get_address():
+        host = settings.DAPR_RUNTIME_HOST
+        if host is None:
+            host = "localhost"
```

Comment:
`host` and `port` will never be `None`, right??

Reply:
yes, removed